core/ledger/snapshotgrpc/mock/peer_ledger.go

```diff
@@ -0,0 +1,1716 @@
+// Code generated by counterfeiter. DO NOT EDIT.
+package mock
```

Comment:
1. We decided to avoid mock within the ledger pkg as much as possible. Are there any difficulties in testing the services using the actual ledger pkg?
2. Even if we need, we need to create a local interface with the exact required methods and then create a mock. For example, why do we need to mock `CommitPvtDataOfOldBlocks()`?

Given that we already have a few mocks to be removed (TODO), I would recommend not to add more mocks.